docs/proposals/pod-pid-namespace.md

```diff
@@ -0,0 +1,61 @@
+# Shared PID Namespace for the Docker Runtime
+
+Pods share many namespaces, but the ability to share a PID namespace was not
+supported by Docker until version 1.12. This document proposes how to roll out
+support for sharing the PID namespace in the docker runtime.
+
+## Motivation
+
+Sharing a PID namespace is discussed in #1615, and enables:
```

Comment:
nit: that link won't work once this is moved to community/develop, as it will have to be.

Either give a full normal link or link to https://issues.k8s.io/1615